packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as OneTimeDeposit } from './OneTimeDeposit.svelte'
+export { default as FundConfirmation } from './FundConfirmation.svelte'
+export { default as AccountCreation } from './AccountCreation.svelte'
```

Comment:
We should probably keep the convention, if you call `InitView` the others maybe should be called `View` too, or none, better check how the rest of the repo is and follow the convention